site/source/doc/api/config.md

```diff
@@ -69,9 +131,8 @@ Defines a new config variable. Options can contain the following fields:
 - `description`: A description of the configuration variable (_required_)
 
 - `scope`: An optional value specifying the scope of the variable. One of
```

Comment:
Do you think this should be changed to specify the valid scopes by pattern? E.g '^$' for global scope and '.+' for non global scope, but also '^file/' for file scopes?